WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelper.kt

```diff
@@ -141,7 +159,10 @@ class PostListItemUiStateHelper @Inject constructor(private val appPrefsWrapper:
             )
             else -> {
                 AppLog.e(POSTS, "MediaError and postError are both null.")
-                null
+                if (BuildConfig.DEBUG) {
```

Comment:
⚠️  Is this intentional? If it is, can we add a message to `IllegalStateException` at least so we'll know what happened?

Reply:
I felt like the message will be in the log right above the exception. But I guess you are right - fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9491/commits/bb2d76ee7cc47c65170a49bc9be9c95a232026be